### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by Oizaro**
Update resource reference and change 'hide element' with a more fast 'remove'.

### CODE REVIEW

File: src/disable-amp.js [hunk 1]
```diff
@@ -12,9 +12,9 @@ function observeDomChanges(callback) {
  * @param amp element
  */
 const hideAmpIcon = (amp) => {
-    const ampIcon = amp.querySelector('[aria-label="AMP logo"]');
+    const ampIcon = amp.querySelector('[aria-label="Logo AMP"]');
     if (ampIcon) {
-        ampIcon.style.display = 'none';
+        ampIcon.remove();
```
Comment (tvinzz) [inline src/disable-amp.js:17]: we don't remove, just hide.

**Comment by Oizaro** [comment]
@tvinzz 👍👍👍

File: src/disable-amp.js [hunk 2]
```diff
@@ -12,9 +12,9 @@ function observeDomChanges(callback) {
  * @param amp element
  */
 const hideAmpIcon = (amp) => {
-    const ampIcon = amp.querySelector('[aria-label="AMP logo"]');
+    const ampIcon = amp.querySelector('[aria-label="Logo AMP"]');
```
Comment (tvinzz) [inline src/disable-amp.js:15]: do you have it as `Logo AMP`?
image

**Comment by Oizaro** [comment]
Yes, on my elements inspector is named "Logo AMP"

**Comment by Oizaro** [comment]
@tvinzz

Ok I have tried with "amp faq" keysearch, and amp logo has appeared again.

All well instead with different keysearh such as "windowsteca" or "windowsblogitalia".

**Pull request closed without merging**